applications/RANSApplication/tests/cpp/k_omega/test_conditions.cpp

```diff
@@ -0,0 +1,225 @@
+//    |  /           |
```

Comment:
I'd call it `test_k_omega_conditions`. Once we merge the apps we can create a subfolder in the cpp tests of the FluidDynamicsApplication called `rans_conditions` or `rans_cpp_tests`.